Proposals/ParametricandOpticalAxes/ProposalSummary.md

```diff
@@ -0,0 +1,373 @@
+
+# Proposal: Parametric and Optical Axes
+
+## Administrative Information
+**Proposers name:** Sam Berlow
+
+**Vendor affiliation:** Type Network
+
+**Proposal name:** Parametric Axes
+
+**Date of submission:** 12/6/2017
+
+**New or revised proposal:** New
+
+**Previous revision date:** N/A
+
+## General Technical Information
```

Comment:
The fields in this section should be completed in this section and apply to all of the axes covered in a single ProposalSummary page.